Amazon.IonObjectMapper.Test/TestObjects.cs

```diff
@@ -234,6 +236,59 @@ public override string ToString()
         }
     }
 
+    public class EngineSerializerFactory : IonSerializerFactory<Engine>
```

Comment:
I think these factories and serializers should move to another file and we keep this file for plain old object.

Reply:
Fixed.